pom.xml

```diff
@@ -361,6 +362,13 @@
                 <version>${postgresql.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
Don't add the connector's dependency in root pom, just use it in connector's pom.xml, reference: https://github.com/apache/incubator-seatunnel/pull/2630/files